egs/gale_arabic/s5b/conf/online_pitch.conf

```diff
@@ -0,0 +1,46 @@
+## This config is given by conf/make_pitch_online.sh to the program compute-and-process-kaldi-pitch-feats,
+## and is copied by steps/online/nnet2/prepare_online_decoding.sh and similar scripts, to be given
```

Comment:
this shouldn't be needed if pitch features aren't being used, and I'd be surprised if they are being used for arabic (search for make_mfcc_pitch in local/)